tools/ogs-tools/AssignMaterialParameter.xml

```diff
@@ -0,0 +1,50 @@
+<tool id="assign_material_parameter" name="Assign Material Parameter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>
+        Assign material parameters to a VTK Unstructured Grid based on a CSV file.
+    </description>
+    <requirements>
+        <requirement type="package" version="5.9.0">paraview</requirement>
```

Comment:
As soon as we have it (the conda package) we need to switch to ogs. For now I would suggest to create an xml macro (see [here](https://github.com/galaxyproject/tools-iuc/blob/79c3c1123428c177bf4a8c38260232810457845d/tools/dada2/macros.xml#L4)) for the requirements that can be used in all tools (and just make it the union of all requirements).